PostReleaseActivities.md

```diff
@@ -26,3 +26,8 @@ Please follow the below steps after publishing analyzer NuGet packages from this
 2. Build.
 3. Ensure that nuget.exe is on path.
 4. Generate notes: Switch to the output directory, say `artifacts\bin\ReleaseNotesUtil\Debug\netcoreapp3.1` and execute `GenDiffNotes.cmd` to generate release notes.  Example command line for v2.9.4 to v2.9.5: `GenDiffNotes.cmd C:\scratch nuget.org 2.9.4 2.9.5`.
+
+## Followup items
+
+1. For `Microsoft.CodeAnalysis.Analyzers` package, update [`MicrosoftCodeAnalysisAnalyzersVersion`](https://github.com/dotnet/roslyn/blob/95809b0b922439465a213922ef7eb81e9b5a223f/eng/Versions.props#L82) in dotnet/roslyn to reference the new package version.
```

Comment:
Would it be possible to replace the exact version with either:
- the [floating versioning](https://docs.microsoft.com/nuget/concepts/package-versioning#floating-version-resolutions)
- the [range versioning](https://docs.microsoft.com/nuget/concepts/package-versioning#version-ranges) 
- some kind of aliasing

This would avoid any manual operation while ensuring the use of the latest.

Reply:
@Evangelink Doing that can cause sudden CI failures if a new released version contained a new analyzer. I don't think this would be intended.

Reply:
I now tend to think that I'd better have roslyn CI crashing because of a new version of roslyn-analyzer than waiting for some release/end-user to find out but then I can understand both sides :)

Reply:
@Evangelink I'll let that up to roslyn team.

cc @RikkiGibson

Reply:
Ideally we would use the Arcade automation to open PRs to roslyn when new versions of roslyn-analyzers come out. e.g. dotnet/roslyn#58859

Reply:
That said, if the way described in this document is accurate for what we actually do, that's fine.